packages/rainbowkit/src/components/ModalSelection/ModalSelection.tsx

```diff
@@ -85,7 +88,19 @@ export const ModalSelection = ({
               src={iconUrl}
               width="28"
             />
-            <div>{name}</div>
+            <Box>
+              <Box style={{ marginTop: recent ? -2 : undefined }}>{name}</Box>
+              {recent && (
+                <Text
+                  color="accentColor"
+                  size="12"
+                  style={{ lineHeight: 1, marginTop: -1 }}
+                  weight="medium"
```

Comment:
couple of inline styles here and line 92. this is because to ensure consistent vertical gaps between all items, the computed height of the button needs to be 28 even if it includes the "recent" label.

however, the label was causing the computed height to be 32 (3px too high).

setting negative margin only in the "recent" label, meant that both texts were too tight together. for this reason, we're offsetting the wallet label by 2, and the recent label by 1. 

<img width="366" alt="image" src="https://user-images.githubusercontent.com/372831/190415164-be7e01c6-6dfa-4e12-9c7b-8a7a45f0656c.png">

Reply:
approved by @mikearndt